Gems/GameStateSamples/Code/Source/GameStateSamplesModule.cpp

```diff
@@ -86,6 +88,22 @@ namespace GameStateSamples
         {
             CryHooksModule::OnCrySystemInitialized(system, systemInitParams);
 
+            AZ::TickBus::Handler::BusConnect();
+        }
+
+        void OnTick([[maybe_unused]]float deltaTime, [[maybe_unused]]AZ::ScriptTimePoint scriptTimePoint) override
```

Comment:
Will revisit the need to wait a tick in order to load a UI Canvas from C++ as a separate issue.